### ISSUE TITLE
Fallback meta description when working with pageblocks (SEO)

### ISSUE THREAD

**Issue by krista-ramakers** [driebit/ginger#264]
We have this nice feature: https://github.com/driebit/ginger/issues/161, but this is not a fallback solution in the cases of websites that only use page blocks to add content (these are used more and more frequent). 
Can we come up with a way to provide these websites with a fallback meta description as well?

**Comment by mworrell** [comment]
Discussed this with @DorienD. My opinion is that, if possible, a resource should have a summary. This helps in communicating the resource via meta-tags, feeds, etc. It is also useful in generic overviews.

In the Mediamatic site we (@fredpook and I) added the property `introduction_html`. This property is copied to the summary when the resource is saved. This helps in sharing the resource between sites.

In short: if you divert from the standard summary/body paradigm, then either provide a way to extract a summary or don't expect to have a summary.
